tools/NuGet/build_props/DynamoVisualProgramming.Tests.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+    <ItemGroup>
+        <Content Include="$(MSBuildThisFileDirectory)TestServices.dll.config">
```

Comment:
when does this props file get used?

Reply:
I am just curious, is there a simpler way to get this config file to be copied - putting it in the lib subfolder folder does not work?

Reply:
Props files (from nugets) are consumed by the project that references the nuget when running restore.
This is a pretty standard way to have both assemblies and generic files in a nuget pacakge. There is another way to add generic files (as contentFIles) but those seem to be readonly (and we need write access).
https://learn.microsoft.com/en-us/nuget/concepts/msbuild-props-and-targets#packagereference-projects